src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -6,541 +6,388 @@
-  //  @Inject private AmazonS3 amazonS3;
-  //
-  //  @Inject
-  //  @Named("nonBufferingRestTemplate")
-  //  private RestTemplate nonBufferingRestTemplate;
-  //
-  //  @Inject
-  //  @Named("transformClientRestTemplate")
-  //  private RestTemplate transformClientRestTemplate;
-  //
-  //  @Value("${endpoints.store.version}")
-  //  private String storeApiVersion;
-  //
-  //  @Value("${endpointUrl.retrieve}")
-  //  private String endpointUrlRetrieve;
-  //
-  //  @Value("${endpointUrl.index}")
-  //  private String endpointUrlIndex;
-  //
-  //  @Value("${endpoints.index.version}")
-  //  private String indexApiVersion;
-  //
-  //  @Value("${endpointUrl.transform}")
-  //  private String endpointUrlTransform;
-  //
-  //  @Value("${endpoints.transform.version}")
-  //  private String endpointsTransformVersion;
-  //
-  //  @Value("${s3.bucket.file}")
-  //  private String fileBucket;
-  //
-  //  @Value("${s3.bucket.irm}")
-  //  private String irmBucket;
-  //
-  //  @Value("${s3.bucket.metacard}")
-  //  private String metacardBucket;
-  //
-  //  private static final MultiValueMap<String, HttpEntity<?>> TEST_INGEST_REQUEST_BODY;
-  //
-  //  static {
-  //    final MultipartBodyBuilder builder = new MultipartBodyBuilder();
-  //    builder.part("file", TEST_FILE_RESOURCE);
-  //    builder.part("metacard", TEST_METACARD_RESOURCE);
-  //    builder.part("correlationId", "000f4e4a");
-  //    TEST_INGEST_REQUEST_BODY = builder.build();
-  //  }
-  //
-  //  @BeforeEach
-  //  public void beforeEach() {
-  //    restTemplate =
-  //        new CustomTestRestTemplate(applicationContext)
-  //            .addRequestHeader(StoreController.ACCEPT_VERSION_HEADER_NAME, storeApiVersion);
-  //
-  //    indexMockRestServiceServer = MockRestServiceServer.createServer(nonBufferingRestTemplate);
-  //    transformMockRestServiceServer =
-  //        MockRestServiceServer.createServer(transformClientRestTemplate);
-  //
-  //    amazonS3.createBucket(fileBucket);
-  //    amazonS3.createBucket(irmBucket);
-  //    amazonS3.createBucket(metacardBucket);
-  //  }
-  //
-  //  @AfterEach
-  //  public void afterEach() {
-  //    indexMockRestServiceServer.verify();
-  //    transformMockRestServiceServer.verify();
-  //    transformMockRestServiceServer.reset();
-  //
-  //    cleanBucket(fileBucket);
-  //    cleanBucket(irmBucket);
-  //    cleanBucket(metacardBucket);
-  //  }
-  //
-  //  private void cleanBucket(String bucket) {
-  //    amazonS3.listObjects(bucket).getObjectSummaries().stream()
-  //        .map(S3ObjectSummary::getKey)
-  //        .forEach(key -> amazonS3.deleteObject(bucket, key));
-  //    amazonS3.deleteBucket(bucket);
-  //  }
-  //
-  //  @Test
-  //  public void testContextLoads() {}
-  //
-  //  /**
-  //   * @see #testRetrieveFileWhenS3IsEmpty()
-  //   * @see StoreControllerRetrieveFileComponentTest#testS3KeyDoesNotExist()
-  //   */
-  //  @Test
-  //  public void testRetrieveFileNotFound() throws Exception {
```

Comment:
These tests that tested retrieving specific content I felt were not longer needed because we only deal in data types now.